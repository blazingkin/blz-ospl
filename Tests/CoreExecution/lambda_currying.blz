:main
    curried_func = (x -> (y -> x + y))
    add_three = curried_func(3)
    print(add_three(5))
    add_pi = curried_func({pi})
    print(add_pi(20))
end