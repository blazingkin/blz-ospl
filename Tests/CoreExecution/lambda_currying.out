8
23.141592653589793
